Source/Core/VideoCommon/BPStructs.cpp

```diff
@@ -753,61 +753,808 @@ void LoadBPRegPreprocess(u32 value0)
   }
 }
 
-void GetBPRegInfo(const u8* data, std::string* name, std::string* desc)
+template <>
+struct fmt::formatter<GenMode>
 {
-  const char* no_yes[2] = {"No", "Yes"};
+  constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+  template <typename FormatContext>
+  auto format(const GenMode& mode, FormatContext& ctx)
+  {
+    const char* cullmodes[] = {"None", "Back-facing primitives only",
+                               "Front-facing primitives only", "All primitives"};
```

Comment:
```suggestion
                               "Front-facing primitives only", "All primitives",
                           };
```

A comma after the last element of the array will make the ending brace get formatted in a more immediately visable manner. Ditto for other relevant occurrences.

Reply:
The ending comma causes it to be formatted like this:

```C++
    const char* cullmodes[] = {
        "None",
        "Back-facing primitives only",
        "Front-facing primitives only",
        "All primitives",
    };
```

I somewhat prefer having it on fewer lines, especially for arrays where each element is only a few characters and it fits on a single line (e.g. `{"None", "S", "T", "ST", "U", "SU", "TU", "STU"}`).  Should I change all of those, or should I leave the single-line ones as-is?

Reply:
I'm fine with keeping one liner arrays, where everything is more or less the same predictable length, I think it's fine to place it on a single line. Ones that have very different string lengths, like `cullmodes` I think should have a comma, since it makes more sense to make those vertically identifiable, so they aren't accidentally scanned over when reading left to right.